src/js/actions/mask.js

```diff
@@ -269,11 +269,16 @@ define(function (require, exports) {
                     coalesce: true,
                     suppressHistoryStateNotification: true
                 }
-            };
+            },
+            createActions = Immutable.List.of({
+                layer: currentLayer,
+                playObject: [
+                    vectorMaskLib.makeBoundsWorkPath(bounds),
+                    vectorMaskLib.makeVectorMaskFromWorkPath(),
+                    vectorMaskLib.deleteWorkPath()]
+            });
 
-        return descriptor.batchPlayObjects([vectorMaskLib.makeBoundsWorkPath(bounds),
-                vectorMaskLib.makeVectorMaskFromWorkPath(),
-                vectorMaskLib.deleteWorkPath()], options)
+        return layerActionsUtil.playLayerActions(currentDocument, createActions, true, options)
```

Comment:
you still have to make a List-Of-One for the layer. But, it is reduces the file by 4 lines so i guess its a win.